packages/@coorpacademy-components/src/atom/cta/index.js

```diff
@@ -18,13 +35,13 @@ class CTA extends React.Component {
   }
 
   handleMouseEnter() {
-    return this.setState(prevState => ({
+    return this.setState(() => ({
```

Comment:
Why even use an updater if we're not depending on previous state, just pass an object

Reply:
I removed unused params. I could do more, but that you propose is not iso. There is a subtle difference between both. Mine is lazy, your's eager.
setState wait in the first place an updater function.
In this case, we don't care, the state isn't too complexe to pay attention of evaluation's performance.